package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "got": "~2.4.0",
-    "json-lint": "~0.1.0",
+    "jju": "^1.2.0",
```

Comment:
Are we using `^` or `~` for deps? have we standardized?

I think that `~` is better